hal/include/queue.h

```diff
@@ -0,0 +1,1277 @@
+/*

```

Comment:
Where'd this file come from? Was it generated by HALCoGen?

Reply:
It was generated by HALCoGen